### PR TITLE
update to docs v3

### LINKED ISSUE #782: Docs v1 Update Checklist
# Module Update Checklist
----
Name of Module: Creating a Git Repository
PR for module update: #792 

This is the checklist for bringing a module up to date with the latest major version of docs.md: v1

## Summary of changes

The biggest changes to docs for this version are extensive updates to the required front matter, including lots of new fields, and the introduction of shared macros to replace standardized sections.

The following sections provide a complete (hopefully!) list of potential changes to modules needed to bring them up to date with the new version of docs.md. 

## Front matter

* [x] The front matter includes all required fields from the new templates. Some of these are new fields and some are not. Use the ["Your front matter checklist"](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/docs.md#your-front-matter-checklist) section of docs.md to get a customized list of the required front matter for a given module. 

Also watch specifically for the following:

* [x] current_version_description is a short, present-tense text fragment without bullets, and gives a learner-centric quick overview of version changes from the learner perspective
* [x] version_history contains 1-3 past versions with bulleted links to commit-specific documents, unless this is an initial version of a module, in which case version_history should say "No previous versions." For detailed instructions on how to fill out the version fields, see the [docs descriptions](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/docs.md#version_history).

The following should **NOT** be included in module front matter: 

- [x] REMOVED: any macros defined in the shared macro files
- [x] REMOVED: `link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css` and `link: https://storage.googleapis.com/chop-dbhi-arcus-education-website-assets/css/styles.css`
- [x] REMOVED: `script: https://kit.fontawesome.com/83b2343bd4.js`
- [x] REMOVED: `script:  https://code.jquery.com/jquery-3.6.0.slim.min.js`

## Macros

* For all modules:
    - [x] Overview is created with the macro, not written
    - [x] Feedback is created with the macro, not written

* For modules with interactive R, Python, or SQL:
    - [NA] Lesson Preparation is created with the relevant macro, not written

* For wrapper modules:
    - [NA] Lesson Preparation is created with the relevant macro, not written
    - [NA] There is a `print_resource` macro for each external resource just after the `lesson_prep_wrapper` macro

## Other changes

* [x] Avoid linking to our own modules in the Additional Resources section at the end of each module
* [x] Check that highlight boxes use the updated style

## Final check

These updates should not change the rendered module much, for the most part. 
To check, pull up the rendered version from before the update and your updated version and look for missing/changed/added content, or anything that's not rendering properly. 
For conveience, paste the liascript links here:

- Original: {paste liascript link for original module}
- Update: {paste liascript link for updated module}

When you're ready to finalize the updated module, do the following in the module front matter:

* [x] Increment `version` for the module. If no meaningful changes were made to the content of the module as it renders for learners, this is a revision. If there were updates to the module content (e.g. in the Additional Resources section), then it's a minor update.
* [x] Update `docs_version`

### LINKED ISSUE #783: Docs v2 Update Checklist
# Module Update Checklist
----
Name of Module: Creating a Git Repository
PR for module update: #792 

This is the checklist for bringing a module up to date with the latest major version of docs.md: v2

## Summary of changes

The only change is that `coding_required` is now a mandatory front matter field. 
It must be either `true` or `false`. 
See [the `coding_required` page in docs](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/docs.md#38) for more details about this field. 

## Front matter

* [x] `coding_required` is either `true` or `false`

## Final check

When you're ready to finalize the updated module, do the following in the module front matter:

* [x] Increment `version` for the module. This is a revision. 
* [x] Update `docs_version` to 2.0.0

### LINKED ISSUE #784: Docs v3 Update Checklist
# Module Update Checklist
----
Name of Module: Creating a Git Repository
PR for module update: #792 

This is the checklist for bringing a module up to date with the latest major version of docs.md: v3

## Summary of changes

An optional front matter field, `collection`, is now available. Check if this module should belong in one or more collections. To learn more, read the [description in docs](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/docs.md#collection).

There is also a new option, `text`, for `data_domain`.

Every module should have at least one of `data_domain`, `data_task` or `collection` populated with something.

## Front matter

* [x] If relevant, `collection` is listed, with one or more collections (separated by commas)
* [x] If relevant, add `text` as the data domain

## Final check

When you're ready to finalize the updated module, do the following in the module front matter:

* [x] Increment `version` for the module. This is a revision. 
* [x] Update `docs_version` to 3.0.0

### PR DESCRIPTION
**Pull request by franzenr**
bring in line with docs v1, v2, and v3.

resolves #782 
resolves #783  
resolves #784

### CODE REVIEW

File: git_creation_and_tracking/git_creation_and_tracking.md
Comment (franzenr) [inline git_creation_and_tracking/git_creation_and_tracking.md:45]: @drelliche Not sure if having two previous sequential modules listed is acceptable or if it will throw a wrench into any of the discovery app stuff!

Comment (franzenr) [reply git_creation_and_tracking/git_creation_and_tracking.md:45]: verbal convo with Elizabeth right now confirmed that this is fine!

**Review by rosemm — CHANGES_REQUESTED** [review]
Looks great! One little typo in the metadata, and then I think we're good to go :)

**Comment by franzenr** [comment]
Wonderful, thanks for the great attention to detail, @rosemm! We should be good to merge this in once it has your official stamp of approval!

**Review by rosemm — APPROVED** [review]
Lgtm!

**Pull request merged**
